src/benchmark/config_parser.py

```diff
@@ -325,8 +329,9 @@ def _device_is_correct(device):
 
 
 class OpenVINOParameters(ParametersMethods):
-    def __init__(self, mode, extension, async_request_count, thread_count, stream_count):
+    def __init__(self, mode, perf_hint, extension, async_request_count, thread_count, stream_count):
```

Comment:
what is perfhint for custom-created launcher?  will old-format config files(without perfHint) be working after these changes?